css/components/toast_notify_snackbar.css

```diff
@@ -0,0 +1,179 @@
+/*
+||Toast , Notify , Snackbar css
+*/
+
+/*
+|| Toast
+*/
+.toast{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-width: 15.625rem;
+    background-color: var(--dark-black);
+    color: var(--mint-white);
+    border-radius: 5px;
+    padding: 0.8rem;
+    position: fixed;
+    font-size: 1rem;
+    z-index: 5;  
+}
+
+.toast-top{
+    top: 1rem;  
+    right: 3rem;
+}
+
+.toast-bottom{ 
+    bottom: 1rem;  
+    right: 3rem;
+}
+
+/*
+|| Notify
+*/
+
+.notify{
+    display: flex;
```

Comment:
you can try to reuse the display flex. It is repeating

Reply:
like as flex class